tools/qemu_args.sh

```diff
@@ -3,9 +3,18 @@
 # SPDX-License-Identifier: MPL-2.0
 
 # This script is used to generate QEMU arguments for OSDK.
-# The positional argument $1 is the scheme.
-# A switch "-ovmf" can be passed as an argument to enable OVMF.
-# The enrivonmental variable VSOCK can be passed as 1 to trigger vsock module.
+# Usage: `qemu_args.sh [1]`
+#  - 1: "normal", "microvm" or "iommu";
+# Other arguments are configured via environmental variables:
+#  - OVMF: "1" or "0";
+#  - NETDEV: "user" or "tap";
+#  - VHOST: "0" or "1";
+#  - SMP: number of CPUs;
+#  - MEM: amount of memory, e.g. "8G".
+
+OVMF=${OVMF:-"1"}
+VHOST=${VHOST:-"0"}
```

Comment:
VHOST options are using `on` and `off`. So the default value may be better `off`.